### PR TITLE
feat(switch): add accessibility

### PR DESCRIPTION
**Pull request by SaniyaGupta**
### What does this implement/fix? Explain your changes.

...

### Does this close any currently open issues?

...

### Any other comments?

...

### Dependent PRs/Commits

...

### Describe breaking changes, if any.

...

### Checklist

Check all those that are applicable and complete.

- [ ] Merged with latest `master` branch

### CODE REVIEW

File: core/components/atoms/switchInput/Switch.tsx [hunk 1]
```diff
@@ -6,7 +6,7 @@ export type Size = 'regular' | 'tiny' | 'large';
 export type Appearance = 'primary' | 'alert' | 'success' | 'warning';
 
 type MouseEvent = React.ChangeEvent<HTMLInputElement>;
-
+type KeyboardEvent = React.KeyboardEvent<HTMLInputElement>;
```
Comment (mukuljainx) [inline core/components/atoms/switchInput/Switch.tsx:11]: please add a new line after and before every type

File: core/components/atoms/switchInput/__stories__/index.story.tsx [hunk 2]
```diff
@@ -19,7 +19,7 @@ export const all = () => {
 
   return (
     <div>
-      <Switch disabled={disabled} checked={checked} size={size} onChange={onChangeHandler} />
+      <Switch aria-label="Switch" disabled={disabled} checked={checked} size={size} onChange={onChangeHandler} />
```
Comment (mukuljainx) [inline core/components/atoms/switchInput/__stories__/index.story.tsx]: Can we add an example with Label too? with `htmlFor` and `id`

```
<Label>Have you been vaccinated?</Label>
<Switch>
```

**Comment by mukuljainx** [comment]
```  return(
    <form>
      <Label htmlFor="vaccination-status">
        Have you been vaccinated ?
      </Label>
      <div>
        <Switch id="vaccination-status" name="vaccination-status" onChange={function(){}} />
      </div>
    </form>
  );
```

**Comment by mukuljainx** [comment]
Add accessibility doc block above the props table

**Review by veekays — APPROVED** [review]

**Pull request merged**